frontend/src/Series/Details/SeriesDetailsLink.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Link from 'Components/Link/Link';
+import Label from 'Components/Label';
+import ClipboardButton from 'Components/Link/ClipboardButton';
+import { kinds, sizes } from 'Helpers/Props';
+import styles from './SeriesDetailsLinks.css';
+
+interface SeriesDetailsLinkProps {
+  url: string;
+  label: string;
+  idValue: string | number;
+  copyTitle?: string;
+}
+
+const SeriesDetailsLink: React.FC<SeriesDetailsLinkProps> = ({
+  url,
+  label,
+  idValue,
+  copyTitle = 'Copy ID'
+}) => (
+  <div className={styles.linkGroup}>
+    <Link className={styles.link} to={url}>
+      <Label className={styles.linkLabel} kind={kinds.INFO} size={sizes.LARGE}>
+        {label}
+      </Label>
+    </Link>
+    <div className={styles.idCopyButtonRow}>
+      <span className={styles.idLabel}>{idValue}</span>
+      <ClipboardButton
```

Comment:
The icon is in the wrong spot after copying. Perhaps because it's expected to be part of a form input?
![image](https://github.com/user-attachments/assets/d32be81d-966d-41fc-b909-19b56c068090)

`ClipboardButton` probably needs to support rendering a different type of button.